docs/getting-started/quick-start.md

```diff
@@ -192,10 +192,10 @@ Your final page should look similar to the one below.
 
 ## Installing CKEditor&nbsp;5 from a ZIP file
 
-If you do not want to build your project using npm, and you cannot rely on the CDN delivery, you can download ready-to-run files with CKEditor&nbsp;5 and all its plugins.
+If you do not want to build your project using npm and cannot rely on the CDN delivery, you can download ready-to-run files with CKEditor&nbsp;5 and all its plugins.
 
 1. <a href="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/zip/ckeditor5-{@var ckeditor5-version}.zip">Download the ZIP archive with the latest CKEditor&nbsp;5 distribution.</a>
-2. Extract the ZIP archive into a dedicated directory inside your project. It is recommended to include the editor version in the directory name to ensure proper cache invalidation once a new version of CKEditor&nbsp;5 is installed.
+2. Include the editor version in the directory name to ensure proper cache invalidation whenever you install a new version of CKEditor&nbsp;5.
```

Comment:
Why did you remove the "Extract ..." part?

Now the instruction says "download" and then "include". It seems incomplete.

Reply:
I know there's "Copy necessary files" later on. But what's the point of "Include the editor version in the directory name" if we didn't ask them to extract the archive yet?

Also: asking them to copy **necessary** files creates a risk of copying to little, changing structure. KISS. \\

1.  Download
2.  Extract to `vendor/`. Make sure to have editor version yadadad
3.  Checkout the sample yadada

Reply:
Good points - I'll change it.